packages/babel-core/src/config/files/configuration.js

```diff
@@ -230,7 +231,21 @@ const readConfigJSON5 = makeStaticFileCache((filepath, content) => {
   try {
     options = json5.parse(content);
   } catch (err) {
-    err.message = `${filepath}: Error while parsing config - ${err.message}`;
+    // Check if the error contains source location informations
+    if (
+      typeof err.lineNumber === "number" ||
+      typeof err.columnNumber === "number"
+    ) {
+      const loc: NodeLocation = {
```

Comment:
This is a Flow error, since `NodeLocation` isn't declared anywhere. Also looks like code-frame doesn't actually have `//@flow` and doesn't export the type at the moment :(

Reply:
I added a flow annotation in the file.